.travis.yml

```diff
@@ -0,0 +1,469 @@
+    #   0) Install travis-ci command-line client.
+    #   1) Login with Roberto's account on anaconda.org
+    #   2) Go to profile settings. (https://anaconda.org/theochem/settings/profile)
+    #   3) Select "Access".
+    #   4) Create a new token (all operations)
+    #   5) Copy the token.
+    #   6) Encrypt the token on the command line
+    #      travis encrypt --pro ANACONDA_TOKEN="[copy-paste-your-anaconda-token]"
+    #      This command asks for a username and password. Roberto's account
+    #      must be used.
+    #   7) Copy-paste the encrypted line below
+    #ANACONDA_TOKEN
+    - secure: {{cookiecutter.anaconda_token}}
+    # Instructions for (re)setting up PYPI_PASSWD, for theochem admins
+    #   0) Install travis-ci command-line client.
+    #   1) Encrypt Roberto's Pypo password on the command line
+    #      travis encrypt --pro PYPI_PASSWD="[copy-paste-pypi-password]"
+    #      This command asks for a username and password. Roberto's account
+    #      must be used.
+    #   2) Copy-paste the encrypted line below
+    #PYPI_PASSWD
+    - secure: {{cookiecutter.pypi_pass}}
+    - PROJECT_NAME=gbasis
+    - GITHUB_REPO_NAME=theochem/gbasis
+    - CONDA_PKG_NAME_PY=gbasis
+    - PYPI_LOGIN=theochem
+
+# TEMPLATED FROM https://raw.githubusercontent.com/theochem/derivcheck/master/.travis.yml
+# ####
+# # EVERYTHING BELOW THIS LINE IS AUTOGENERATED.
+# ####
+
+# Do not use Travis Python to save some time.
+language: generic
+os:
+  - linux
+#  - osx
+#osx_image: xcode8.3
+dist: trusty
+sudo: false
+
+#matrix:
+#  # We only test on Linux with one specific Python version, unless we make a release. All
+#  # other cases are allowed to fail and will exit early. With the fast_finish option,
+#  # travis will send a status update to github as soon as the non-allowed-to-fail has
+#  # finished. This speeds up testing in PRs.
+#  #
+#  # This is not ideal. It would be better to run just one item from the build matrix when
+#  # not preparing a release. This is not possible on Travis. There are several tickets on
+#  # travis-ci related to this limitation.
+#  #
+#  #    https://github.com/travis-ci/travis-ci/issues/7451
+#  #    https://github.com/travis-ci/travis-ci/issues/7149
+#  #    https://github.com/travis-ci/travis-ci/issues/2778
+#  #    ...
+#  fast_finish: true
+#  allow_failures:
+#  - os: osx
+
+
+cache:
+  directories:
+    - $HOME/download # Sufficient to add miniconda.sh to TRAVIS cache.
+    - $HOME/miniconda # Add the installation to TRAVIS cache.
+
+before_cache:
+- if ! [[ $TRAVIS_TAG ]]; then rm -rf $HOME/miniconda/conda-bld; fi
+- rm -rf $HOME/miniconda/locks $HOME/miniconda/pkgs $HOME/miniconda/var
+- pip uninstall -y cardboardlint # Cardboardlint always installs even if no changes are made.
+
+branches:
+  only:
+    - master
+    - /^[0-9]+\.[0-9]+(\.[0-9]+)?([ab][0-9]+)?$/
+
+before_install:
+#- if ! [[ $TRAVIS_TAG || $TRAVIS_OS_NAME == "linux" && $MYCONDAPY == "2.7" ]]; then exit 0; fi
```

Comment:
I'm not sure why this line is commented? I would suggest to uncomment it and to change "2.7" to "3.6".

Reply:
P.S. That may also be something to fix in the cookiecutter template?